src/components/pages/dashboard-pages/EditProfile.jsx

```diff
@@ -0,0 +1,20 @@
+import EditProfileForm from "./EditProfileForm";
+import subjectOptions from "../../../data/subjects";
+import { useAuth } from "../../../AuthProvider";
+import { useNavigate } from "react-router-dom";
+
+const EditProfile = () => {
+  const {userData} = useAuth();
+  const navigate = useNavigate();
+  const options = subjectOptions;
+  
+  const cancelEditing = () => navigate("/dashboard");
+    
+    return (
+      <>
```

Comment:
No need for fragments since you are only rendering on element.

Reply:
ok. I'll fix it.